### ISSUE TITLE
Emissions calculation algorithm in data model?

### ISSUE THREAD

**Issue by yeozy95** [Open-Earth-Foundation/OpenClimate-Schema#12]
How does the `EmissionsBreakdown` table treat different algorithms used when calculating emissions? 

Question credits: Gavin Starks, Icebreaker One

**Comment by evanp** [comment]
Wouldn't that be a "methodology"?

**Comment by yeozy95** [comment]
Agreed. I think the calculation of the emissions itself can still be decomposed to `activity_value` * `emissions_factor` - what does into `activity_value` is determined by the "algorithm", and can be detailed in the `Methodologies` table
